SPECS/kata-packages-uvm/kata-packages-uvm.spec

```diff
@@ -42,6 +35,20 @@ Requires:       zlib
 %description
 Metapackage to install the set of packages inside a Kata containers UVM
 
+%package        debug
+Summary:        Metapackage to install the set of packages inside a Kata confidential containers debug UVM.
+Requires:       %{name} = %{version}-%{release}
+Requires:       bash
+Requires:       curl
+Requires:       cpio
+Requires:       grep
+Requires:       readline
+Requires:       gzip
+Requires:       procps-ng
```

Comment:
Have we tested that with removal of these packages from the release image, all tests still pass?

If so, can we quantify savings? This can be interesting metrics